swupd.bash

```diff
@@ -23,7 +23,7 @@ _swupd()
     # $1 is the command being completed, $2 is the current word being expanded
     local opts IFS=$' \t\n'
     local -i i installed
-    local global="--help --url --contenturl --versionurl --port --path --format --nosigcheck --ignore-time --statedir --certpath  --time --no-scripts --no-boot-update --max-parallel-downloads --max-retries --retry-delay --json-output  --allow-insecure-http --debug --quiet --no-progress "
+    local global="--hlp --url --contenturl --versionurl --port --path --format --nosigcheck --ignore-time --statedir --certpath  --time --no-scripts --no-boot-update --max-parallel-downloads --max-retries --retry-delay --json-output  --allow-insecure-http --debug --quiet --no-progress --wait-for-scripts "
```

Comment:
I think there's a typo here:

    --hlp?

Reply:
btw, github++ for showing that. I wouldn't see that in a git diff

Reply:
ups, I must have pressed the delete key without noticing it. Good catch

Reply:
fixed 😊